### ISSUE TITLE
Beware! This tool is not working at all

### ISSUE THREAD

**Issue by koxu1996** [MasterBugPatch/JDeobT#3]
I would like to warn anyone against the use of this tool, because it's shitty. It does not work out-of-box and you need to:
- manually add dependencies and build project (maven or ant would be great)
- fix wrong parsing of current path https://github.com/Ruegg/JDeobT/issues/2
- manually add JAD (there is no mention about it and there are no errors when it doesn't exist, but you need to deduce from code that's used)
- fix using dirs after removing them
- fix using dirs which not exist at all (xxx-Assem,xxx-Deob)
- fix wrong unescaping of obfuscated string (problematic StringEscapeUtils.unescapeJava())
- fix incorrect searching for decryption key (no break after found of fifth value)
and few more...

Instead I reccomend using https://github.com/java-deobfuscator/deobfuscator

**Comment by Ruegg** [comment]
Yeah I agree, I mean I did make this tool when I was 14 years old over two years ago. I don't think anyone is using it though, so I haven't bothered rewriting it.
